InfluxDB 2.0.4 (#2134)
influxdata/docs-v2: content/influxdb/v2.0/reference/cli/influxd/inspect/export-lp.md

| `-h` | `--help`        | Help for the `export-lp` command.                      |            |
|      | `--log-level`   | Log-level (`debug`, `info` _(default)_, or `error`)    | string     |
|      | `--measurement` | Measurement name(s) to export                          | strings    |
|      | `--output-path` | ({{< req >}}) Output file path                         | string     |
|      | `--start`       | Start time to export (RFC3339 format)                  | string     |

## Examples

##### Export all data in a bucket as line protocol
```sh
influxd inspect export-lp \
  --bucket-id 12ab34cd56ef \
  --engine-path ~/.influxdbv2/engine \
  --output-path path/to/export.lp
```

##### Export data in measurements as line protocol
```sh
# Export a single measurement
influxd inspect export-lp \
  --bucket-id 12ab34cd56ef \
  --engine-path ~/.influxdbv2/engine \
  --measurement example-measurement \
  --output-path path/to/export.lp

# Export multiple measurements
influxd inspect export-lp \
  --bucket-id 12ab34cd56ef \
  --engine-path ~/.influxdbv2/engine \
  --measurement example-measurement-1 example-measurement-2 \
  --output-path path/to/export.lp
```

> ##### Export data within time bounds as line protocol

Reviewer: ```suggestion
##### Export data in specified time range as line protocol
```
